Adds a default target and introduces a few variables so that files are installed into $target and not into Python's domain and into /usr.

Signed-off-by: Lucas C. Villa Real <lucasvr@gobolinux.org>
--- xqed-1.0.3/Makefile.orig	2008-03-20 14:22:49.000000000 -0700
+++ xqed-1.0.3/Makefile	2008-03-20 14:24:54.000000000 -0700
@@ -1,13 +1,18 @@
-BIN=/usr/bin
+DESTDIR=/usr
+BINDIR=$(DESTDIR)/bin
+DATADIR=$(DESTDIR)/share/qed
+
+all:
+	@true
 
 install: Qed.py Xqed.py xqed setup.py
-	python setup.py install
-	cp qed xqed $(BIN)
-	-@ln -s $(BIN)/xqed $(BIN)/angolul
-	-@ln -s $(BIN)/xqed $(BIN)/magyarul
-	-@mkdir /usr/share/qed
-	cp eng-hu.dic eng-hu.index pro.help use.help /usr/share/qed
-	cp xqed.xbm xqed.gif xqed-bg.gif xqed-local.gif xqed-idiom.gif xqed-window.gif pronouncing /usr/share/qed
+	python setup.py install --prefix $(DESTDIR)
+	cp qed xqed $(BINDIR)
+	-@ln -s $(BINDIR)/xqed $(BINDIR)/angolul
+	-@ln -s $(BINDIR)/xqed $(BINDIR)/magyarul
+	-@mkdir $(DATADIR)
+	cp eng-hu.dic eng-hu.index pro.help use.help $(DATADIR)
+	cp xqed.xbm xqed.gif xqed-bg.gif xqed-local.gif xqed-idiom.gif xqed-window.gif pronouncing $(DATADIR)
 
 clean:
 	-@rm -fr build
